### DISCUSSION TITLE
Nobelium的原版主题是没有继承过来吗？

### DISCUSSION THREAD

**Discussion by lostornot** [tangly1024/NotionNext#592] [Q&A]
如题

**Comment by tangly1024** [comment]
如您所见😂

**Reply by lostornot** [reply]
😄有考虑加上吗亲亲，是心动的皮肤

**Comment by tangly1024** [comment]
安排在未来的计划中 https://github.com/tangly1024/NotionNext/issues/601

**Comment by tangly1024** [comment]
Nobelium主题已加入项目，后续版本将继续完善。
- https://github.com/tangly1024/NotionNext/issues/601
- https://preview.tangly1024.com/?theme=nobelium

**Reply by lostornot** [reply]
感谢作者😘😘

**Reply by est7** [reply]
Nobelium主题full width下,代码块显示不合适
![image](https://user-images.githubusercontent.com/17064360/221483022-ed80a068-739f-4fac-8069-488b5e27002f.png)
